php/example_code/lambda/Readme.md

```diff
@@ -0,0 +1,98 @@
+# AWS Lambda code examples for the SDK for PHP
```

Comment:
I think just Lambda (no AWS) is preferred in the title.

Reply:
+1

Reply:
fixed